doc/guide/cockpit-session.xml

```diff
@@ -58,6 +58,32 @@ promise.then(user => { ... });
     </variablelist>
 
     <para>Returns a promise that completes once the user information is available.</para>
+    <warning>
+	    <para>
+		    <code>cockpit.user()</code> is soft-deprecated since Cockpit 336, <link
+		    linkend="cockpit-init">cockpit.init()</link> can be used
+		    instead to obtain the user on initial page load.
+
+		    If your page needs to maintain compatibility with
+		    older Cockpit versions you can fallback to
+		    <code>cockpit.user()</code> like the example below:
```

Comment:
Are these.. tabs??!!

Reply:
Yeah, that's what my editor does with XML and imo feels right?

Reply:
It's way over-indented; we use 2-space indentation everywhere else. (We should really find some time to convert to asciidoc :wink: )